docs/content/routing/providers/kubernetes-crd.md

```diff
@@ -1645,8 +1645,8 @@ or referencing TLS stores in the [`IngressRoute`](#kind-ingressroute) / [`Ingres
     kind: TLSStore
     metadata:
       name: default
-      namespace: default
-    
+      namespace: traefik
```

Comment:
Why do you think we should change the namespace? I would remove it at most, but the `traefik` namespace is not shared by the resources below.

Reply:
Yes 👍